services/121-service/src/programs/dto/create-program.dto.ts

```diff
@@ -57,11 +85,11 @@ export class CreateProgramDto {
   })
   public readonly currency: string;
 
-  @ApiProperty()
+  @ApiProperty({ example: 'week' })
```

Comment:
We could possibly also use something like?

```suggestion
  @ApiProperty({
    example: 'week',
    enum: ['week', 'month'],
  })
```